src/pageManager.js

```diff
@@ -521,8 +520,6 @@ export async function initialize() {
             // committed. See: https://github.com/mdn/content/issues/4469
             webNavigationTimeStamp: details.timeStamp
         });
-    }, {
```

Comment:
This filter is supported in Chrome, but [per the docs](https://developer.chrome.com/docs/extensions/reference/webNavigation/#event-onHistoryStateUpdated):

> Conditions that the URL being navigated to must satisfy. The 'schemes' and 'ports' fields of UrlFilter are ignored for this event.